packages/core/src/rest/api-request-builder.ts

```diff
@@ -0,0 +1,14 @@
+import { RestRequestBuilder } from './rest-request-builder';
+
+export type ParametersType<
```

Comment:
Where are these types used. I do not see a reference

Reply:
Yeah, at least the second one might not be necessary, I'll check.

Reply:
I removed the second one and moved the structures a bit to separate better.